DevTools/HackStudio/main.cpp

```diff
@@ -190,12 +193,14 @@ int main(int argc, char** argv)
     if (!make_is_available())
         GUI::MessageBox::show(g_window, "The 'make' command is not available. You probably want to install the binutils, gcc, and make ports from the root of the Serenity repository.", "Error", GUI::MessageBox::Type::Error);
 
-    String argument_absolute_path;
-    if (argc >= 2)
-        argument_absolute_path = Core::File::real_path_for(argv[1]);
+    const char* workspace_argument = nullptr;
+    Core::ArgsParser args_parser;
+    args_parser.add_positional_argument(workspace_argument, "Path to workspace", "workspace", Core::ArgsParser::Required::No);
+    args_parser.parse(argc, argv);
 
-    if (!argument_absolute_path.is_empty() && argument_absolute_path.ends_with(".hackstudio"))
-        open_project(argument_absolute_path);
+    auto workspace = Core::File::real_path_for(workspace_argument);
+    if (workspace.ends_with(".hackstudio"))
```

Comment:
why no `!workspace.is_empty()` check here?

Reply:
Since [`ends_with`](https://github.com/SerenityOS/serenity/blob/master/AK/StringUtils.cpp#L186) already checks for `is_empty`.

Reply:
makes sense.